src/components/About/Terms.js

```diff
@@ -4,9 +4,15 @@ import './About.scss'
 
 import {Typography} from '@material-ui/core';
 
+import Bar from '../BottomBar/BottomBar'
 function Terms() {
-
+    const bar = {
```

Comment:
As I can see **Bar** is the same for three components so far. Make one parent component which will define it ONLY once. It looks like this should be part of main App.js if this is something like NavBar or Header or Footer.

Anytime you have repeating code think about moving it someplace common.